src/manageiq_client/api.py

```diff
@@ -230,7 +236,7 @@ def reload(self, expand=False):
             kwargs = {"expand": expand}
         else:
             kwargs = {}
-        self._data = self._api.get(self._href, **kwargs)
+        self._data = self._api.get(self._href, save_response=False, **kwargs)
```

Comment:
Can you explain this one, and the one in `Entity.reload()`?
I see `reload()` are called implicitly from many places, I suppose you don't want `api.response` changed unpredictably?
OTOH there are things like `entity.exists()` that "obviously" make a request and not saving the response might be surprising.

What I mainly wonder, is this a "slightly nicer to use this way" thing, or an essential "would break otherwise" thing?
If nicer, LGTM.  If it's essential, that'd be a sign this approach is brittle and I'd prefer something more explicit— it seems very hard to me to follow all code paths and which response would be saved last in every case...